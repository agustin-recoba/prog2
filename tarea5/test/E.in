encolar 2
encolar 3
encolar 4
encolar 6
estaVacioAvl
arregloAAvl 4 10 12 13 14
unionDeConjuntos 2 3 6
interseccionDeConjuntos 4 2 3 4 6
estaLlenaPila
estaVaciaPila
frente
desencolar
derAvl
alturaDeAvl
perteneceAConjunto 2
perteneceAConjunto 6
iteradorDeConjunto
Fin
