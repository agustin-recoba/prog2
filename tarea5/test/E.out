1>Encolado.
2>Encolado.
3>Encolado.
4>Encolado.
5>Vacio.
6>
--(14,0.00)
-(13,0.00)
(12,0.00)
-(10,0.00)
7>Union.
8>Intersección.
9>p NO está llena.
10>p está vacia.
11>Binario vacío.
12>Desencolado.
13>13
14>Altura de avl: 3.
15>No pertenece.
16>Pertenece.
17>3 6 
18>Fin.
